wnprc_billing/resources/queries/ehr/project.query.xml

```diff
@@ -9,6 +9,7 @@
                             <fkDbSchema>ehr</fkDbSchema>
                             <fkTable>investigators</fkTable>
                             <fkColumnName>rowid</fkColumnName>
+                            <fkDisplayColumnName>lastName</fkDisplayColumnName>
```

Comment:
Is this OK, or do we need to inject a calculated column that does magic with lastName, firstName, and displayName like we're doing in aliasesWithInvestigators?

Reply:
Yes, injecting a magic calculated column into Investigators seems better so we can use it both here and in aliasesWithInvestigators and projectsWithInvestigators.

Reply:
Did this by having a new investigatorsWithName query instead of doing it in a customizer.